src/components/pages/Faucet/Faucet.tsx

```diff
@@ -101,41 +106,68 @@ export const Faucet = () => {
     const handleContractTransaction = useHandleContractTransaction();
     const { address: account } = useAccount();
     const { data: client } = useWalletClient();
+    const chainId = useChainId();
     const sf = useSF();
 
     const { data: currencies } = useCurrencies();
-    const assetList = toOptions(currencies, CurrencySymbol.USDC).filter(
-        ccy =>
-            currencyMap[ccy.value].toCurrency().isToken &&
-            ccy.label !== CurrencySymbol.USDFC &&
-            ccy.label !== CurrencySymbol.JPYC
-    );
+    const assetList = useMemo(() => {
+        const list = toOptions(currencies, CurrencySymbol.USDC).filter(
+            ccy =>
+                currencyMap[ccy.value].toCurrency().isToken &&
+                ccy.label !== CurrencySymbol.USDFC
+        );
+
+        if (chainId === 11155111) {
+            return list.sort((a, b) => {
+                if (a.value === CurrencySymbol.JPYC) return -1;
+                if (b.value === CurrencySymbol.JPYC) return 1;
+                return 0;
+            });
+        }
```

Comment:
JPYC might be added for other chains in the future.
So `chainId` should not be used here.